### ISSUE TITLE
Is it possible to use ML.NET to implement the AI part?

### ISSUE THREAD

**Issue by ElderJames** [Microsoft/ailab#9]

**Comment by tarasha** [comment]
Wow, that's a great suggestion. We haven't explored that path yet. Do you have suggested methods to make it work using ML.net?

**Comment by ElderJames** [comment]
I'm not clearly,but I think it's a good idea

**Comment by jusso-dev** [comment]
ML.NET doesn't handle CNNs to do object detection per-say but might be possible using a RNN or just NN .. doubtful tho..

**Comment by tarasha** [comment]
Yes, this needs more in-depth analysis and we can comment further based on that. Stay tuned!
